src/wizard/capacitor/index.md

```diff
@@ -0,0 +1,87 @@
+---
+name: Capacitor
+doc_link: https://docs.sentry.io/platforms/capacitor/
+support_level: production
+type: framework
+---
+
+## Integrating the SDK
+
+Sentry captures data by using an SDK within your application’s runtime. These are platform-specific and allow Sentry to have a deep understanding of how your app works.
+
+The Capacitor SDK is currently available for Android devices.
+To install the Capacitor SDK, use npm or yarn:
+
+```bash {tabTitle:npm}
```

Comment:
There's no tab support on the wizard, unfortunately.
So these two items will render one on top of the other and {tabTitle} won't do anything